cli/src/utils/getdeps.ts

```diff
@@ -38,10 +44,26 @@ export async function getDependencies(projectDir: string) {
     if (p === '@reshuffle/code-transform') continue;
     // exclude already visited deps
     if (dependencies.has(p)) continue;
-    dependencies.add(p);
-    for (const subpackage of Object.keys(lockDeps[p].requires || {})) {
-      toProcess.push(subpackage);
+
+    if (lockHasProperty(p)) {
+      dependencies.add(p);
+      toProcess.push(...recurseRequires(lockDeps[p]));
+    } else {
+      // tslint:disable-next-line:no-console
+      console.error(`WARN: Cannot find dependency ${p} in package-lock.json, skipping upload`);
     }
   }
   return dependencies;
 }
+
+function recurseRequires(scope: PackageScope) {
```

Comment:
Maybe call this `recurseDependencies` since we are recursing on the dependencies?

Reply:
I thought about it, but since it gets the attribute `requires` recursively I decided to reflect that in the name.
Not critical.

Reply:
Renamed to `deepRequires`.